deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Datadog.Trace.ClrProfiler.WindowsInstaller.wixproj

```diff
@@ -17,8 +17,8 @@
     <!-- Custom InstallerVersion property for the generated installer,
          value is set to build constant InstallerVersion below and
          accessed as $(var.InstallerVersion) from WiX files.
-         This is the default value is one is not specified when building. -->
-    <InstallerVersion Condition="'$(InstallerVersion)'==''">0.2.2</InstallerVersion>
+         This is the default value if one is not specified when building. -->
+    <InstallerVersion Condition="'$(InstallerVersion)'==''">0.1</InstallerVersion>
```

Comment:
I'm intentionally making this one very low and not updating it again so hand-built versions don't conflict with real releases.